src/gaxios.ts

```diff
@@ -271,7 +271,7 @@ export class Gaxios {
     }
 
     opts.validateStatus = opts.validateStatus || this.validateStatus;
-    opts.responseType = opts.responseType || 'json';
+    opts.responseType = opts.responseType || 'unknown';
```

Comment:
I want to get some opinions on whether or not we would consider this a breaking change. Previously we defaulted for to JSON as the `contentType` if the caller did not specify one. This is sort of a faulty assumption and we probably should have been looking at the response headers. I changed this to default to unknown but I fear that consumers may have been relying on this old JSON behavior.

Reply:
I think this would technically be breaking, although customers would be relying on a bug.

Reply:
If we do this, we should also update the README: https://github.com/googleapis/gaxios#request-options